docs/source/en/glossary.mdx

```diff
@@ -282,21 +302,27 @@ A task that combines texts with another kind of inputs (for instance images).
 
 ## N
 
-### Natural language generation
+### Natural language generation (NLG)
 
 All tasks related to generating text (for instance talk with transformers, translation).
```

Comment:
Did you mean Write With Transformers?

```suggestion
All tasks related to generating text (for instance, [Write With Transformers](https://transformer.huggingface.co/), translation).
```

Reply:
I didn't write this entry but good catch!